airtime_mvc/public/js/airtime/library/library.js

```diff
@@ -942,6 +942,25 @@ var AIRTIME = (function(AIRTIME) {
 
         var simpleSearchText;
 
+        /* set focus into the input box when page loads */
+        $(window).ready(function(){
+            $libContent.find("#library_display_filter label").find('input').focus();
+        } );
+        /* if the user clicks on an a anchor link ie most navigation buttons or there is no focus
+         * then bring focus back to the search window
+         * this should handle most navigation cases for the site currently without taking focus away from
+         * other input fields or otherwise interfering with the UI interaction */
+        $(window).click(function(){
```

Comment:
i'm wondering if we could simplify this to just use click events on the "showbuilder" main menu links. if we add the event to a container div rather than the link then pretty sure we won't interrupt anything, just take advantage of the event "bubbling" up:
```
        $(".media_type_selector").click(function() {
            $libContent.find("#library_display_filter label").find('input').focus();
        });
```